.github/workflows/unit.yml

```diff
@@ -7,11 +7,23 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 8.0, 8.1]
-                symfony: [4.4.*, 5.4.*, 6.0.*]
+                php: [7.4, 8.0, 8.1, 8.2]
+                symfony: [4.4.*, 5.4.*, 6.0.*, 6.1.*, 6.2.*, 6.3.*]
```

Comment:
please remove all those `6.*` jobs for unmaintained versions.

to me, we should have normal jobs running without a special symfony requirement, and then extra included jobs only for LTS versions (`4.4.*` and `5.4.*`) which would use Flex to apply the restriction (and would be added for only 1 PHP version as we don't need to test all combinations)

Reply:
I was tempting to do that, then changed my mind by adding all the version until sf6.4 and 7.0 releases.
That's good for me, I change it.